### PR TITLE
Count datasets with at least one datafile instead of counting datasets marked as published

### PR DESCRIPTION
**Pull request by laurentqro**
screen shot 2018-02-09 at 2 43 33 pm

### CODE REVIEW

**Comment by maxf** [comment]
@laurentqro  What's the 'published' flag used for?

**Comment by laurentqro** [comment]
It's used as the opposite of `draft`

File: app/views/home/dashboard.html.erb [hunk 1]
```diff
@@ -10,21 +10,21 @@
...
-      <span class="data-item bold-xsmall">published datasets</span>
+      <span class="data-item bold-xsmall">datasets published with datafiles</span>
     </div>
   </div>
 
   <div class="column-one-third">
     <div class="data">
       <span class="data-item bold-xlarge"><%= number_with_delimiter(@unpublished_datasets_count) %></span>
-      <span class="data-item bold-xsmall">datasets with no datafiles</span>
+      <span class="data-item bold-xsmall">datasets published with no datafiles</span>
```
Comment (andrewgarner) [inline app/views/home/dashboard.html.erb]: This is confusing. The variable says `@unpublished_datasets_` but the suffix says "datasets published".

Comment (laurentqro) [reply app/views/home/dashboard.html.erb]: I agree, "unpublished" is an awful name for a dataset published with no datafile, but apparently it's what everyone calls it in the legacy system. ** sigh **

I am happy to just say "published with no datafiles" everywhere. I'll make the change.

Comment (andrewgarner) [reply app/views/home/dashboard.html.erb]: The variable name implies it is a count of datasets that have not been published. However, the suffix says "datasets published".

Comment (andrewgarner) [reply app/views/home/dashboard.html.erb]: Cool! I guess we should try and introduce a domain language that's easier to understand in this brave new world :)

**Pull request merged**